packages/block-library/src/footnotes/index.php

```diff
@@ -77,6 +72,24 @@ function register_block_core_footnotes() {
 }
 add_action( 'init', 'register_block_core_footnotes' );
 
+/**
+ * Filters blocks to rewrite footnote links to be popovers instead.
+ *
+ * @param string $block_content Block content.
+ * @return string Filtered block content.
+ */
+function filter_blocks_to_rewrite_footnote_links( $block_content ) {
```

Comment:
- [ ] How does this handle nested blocks? Will they end up getting processed multiple times?

Reply:
This should perhaps be done on the `the_content` filter instead of `render_block`. Also, if there is a block attribute for whether popover is used, this would allow us to access that.